turbinia/client.py

```diff
@@ -867,6 +867,15 @@ def __init__(self, jobs_blacklist=None, jobs_whitelist=None):
       jobs_whitelist (Optional[list[str]]): The only Jobs we will include to run
     """
     config.LoadConfig()
+    if jobs_whitelist or jobs_blacklist:
```

Comment:
What do you think about moving this to `task_manager.setup()` since that's where the other job filtering happens, and it would be nice to keep the related code together: https://github.com/google/turbinia/blob/1e168387c701cb1abe0be3448871b9ca32d833c5/turbinia/task_manager.py#L133

Reply:
Done. Also changed the error message.